host-spec/c06-consensus.tm

```diff
@@ -351,25 +456,20 @@
   </definition>
 
   <\algorithm>
-    <label|algo-slot-time><name|Slot-Time>(<math|s>: the slot number of the
-    slot whose time needs to be determined)
+    <label|algo-slot-time><name|Median-Algorithm>(<math|\<frak-E\><rsub|j>>:
```

Comment:
We should say that the median algorithm has to be run at the beginning of a sync epoch

Reply:
Added.